app/views/play/level/modal/CourseVictoryComponent.vue

```diff
@@ -129,11 +141,16 @@
             link = "/play/level/#{@nextLevel.slug}?course=#{@course._id}&course-instance=#{@courseInstanceID}"
             link += "&codeLanguage=" + @level.primerLanguage if @level.primerLanguage
           return link
+      nextLevelVideoLink: ->
+        link = "/play/video/level/#{@nextLevel.slug}?course=#{@course._id}&course-instance=#{@courseInstanceID}"
+        link += "&codeLanguage=" + @level.primerLanguage if @level.primerLanguage
+        link += "&level=" + @nextLevel.original
+        return link
       nextLevelLinkClasses: ->
```

Comment:
Seems kinda weird to put class selection in computed rather than the template directly.  No need to change this now.